### ISSUE TITLE
Vale.Terms is flagging properly cased products as incorrect

### ISSUE THREAD

**Issue by nickvigilante** [jdkato/txtlint#545]
Vale version 2.21.3
This reproduced both in the Vale action and on my local Mac.

When running Vale against [this file](https://github.com/cockroachdb/docs/blob/master/v22.2/connect-to-the-database.md) in our repo, the following items were flagged:

```
 v22.2/connect-to-the-database.md
 33:63   error    Use 'javascript' instead of 'JavaScript'.                                     Vale.Terms
 148:44  error    Use 'sequelize' instead of 'Sequelize'.                                       Vale.Terms
 148:73  error    Use 'sequelize' instead of 'Sequelize'.                                       Vale.Terms
 538:44  error    Use 'sqlalchemy' instead of 'SQLAlchemy'.                                     Vale.Terms
 538:74  error    Use 'sqlalchemy' instead of 'SQLAlchemy'.                                     Vale.Terms
```

It's easy for us to work around this, but it's odd they got flagged in the first place.

**Comment by jdkato** [comment]
This is working as intended. 

The entries in `Vocab/CockroachDB/accept.txt` determine what is or isn't "properly cased" and you have both `javascript` and `sequelize` listed. If you intend to only use this file to specify spelling exceptions, you can disable `Vale.Terms`. Otherwise, you'll need to change the case of these entries.

**Comment by nickvigilante** [comment]
Thanks for clarifying. I vaguely remember the old way terms was handled, it was case-insensitive, so this is good to know. I've made updates to the repo.
